.github/workflows/CI.yml

```diff
@@ -0,0 +1,26 @@
+name: conitnuous integration Github Actions
```

Comment:
It could be better to split linters (static test) and tests with Jest (dynamic test) in two files.